iOS/DuckDuckGo/DebugScreensViewModel+Screens.swift

```diff
@@ -0,0 +1,187 @@
+            }),
+            .view(title: "Crashes", { _ in
+                CrashDebugScreen()
+            }),
+            .view(title: "New Tab Page", { _ in
+                NewTabPageSectionsDebugView()
+            }),
+            .view(title: "WebView State Restoration", { _ in
+                WebViewStateRestorationDebugView()
+            }),
+            .view(title: "History", { _ in
+                HistoryDebugRootView()
+            }),
+            .view(title: "Bookmarks", { _ in
+                BookmarksDebugRootView()
+            }),
+            .view(title: "Remote Messaging", { _ in
+                RemoteMessagingDebugRootView()
+            }),
+            .view(title: "Vanilla Web View", { d in
+                let configuration = WKWebViewConfiguration()
+                configuration.websiteDataStore = WKWebsiteDataStore.nonPersistent()
+                configuration.processPool = WKProcessPool()
+
+                let ddgURL = URL(string: "https://duckduckgo.com/")!
+                let tab = d.tabManager.model.safeGetTabAt(d.tabManager.model.currentIndex)
+                let url = tab?.link?.url ?? ddgURL
+                return BareBonesBrowserView(initialURL: url,
+                                            homeURL: ddgURL,
+                                            uiDelegate: nil,
+                                            configuration: configuration,
+                                            userAgent: DefaultUserAgentManager.duckDuckGoUserAgent)
+
+            }),
+
+            // MARK: Controllers
+            .controller(title: "Image Cache", { d in
+                let storyboard = UIStoryboard(name: "Debug", bundle: nil)
+                return storyboard.instantiateViewController(identifier: "ImageCacheDebugViewController") { coder in
+                    ImageCacheDebugViewController(coder: coder,
+                                                  bookmarksDatabase: d.bookmarksDatabase,
+                                                  fireproofing: d.fireproofing)
+                }
+            }),
+            .controller(title: "Sync", { d in
+                let storyboard = UIStoryboard(name: "Debug", bundle: nil)
+                return storyboard.instantiateViewController(identifier: "SyncDebugViewController") { coder in
+                    SyncDebugViewController(coder: coder,
+                                            sync: d.syncService,
+                                            bookmarksDatabase: d.bookmarksDatabase)
+                }
+            }),
+            .controller(title: "Configuration Refresh Info", { _ in
+                let storyboard = UIStoryboard(name: "Debug", bundle: nil)
+                return storyboard.instantiateViewController(identifier: "ConfigurationDebugViewController") { coder in
+                    ConfigurationDebugViewController(coder: coder)
+                }
+            }),
+            .controller(title: "NetP", { _ in
```

Comment:
We could rename this to `VPN` now while we're making changes, but definitely not a blocker.